packages/dx-react-scheduler-material-ui/src/templates/common/container.test.jsx

```diff
@@ -21,16 +21,7 @@ describe('Common', () => {
       expect(tree.find(`.${defaultProps.classes.container}`))
         .toBeTruthy();
     });
-    it('should pass rest props to the root element', () => {
```

Comment:
Why you remove the test?

Reply:
I thought that he was no longer needed, but there were changes, it's time to return it)Thank you)